### PR TITLE
scope function on sched.kt kotlincleanup

### PR DESCRIPTION
**Pull request by criticalAY**
## Pull Request template
KotlinCleanup

## Purpose / Description
Scope function at desired places

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.kt [hunk 1]
```diff
@@ -1919,19 +1921,19 @@ open class SchedV2(col: Collection) : AbstractSched(col) {
...
-        dict.put("ints", oconf.getJSONObject("new").getJSONArray("ints"))
-        dict.put("initialFactor", oconf.getJSONObject("new").getInt("initialFactor"))
-        dict.put("bury", oconf.getJSONObject("new").optBoolean("bury", true))
-        dict.put("delays", oconf.getJSONObject("new").getJSONArray("delays"))
-        // overrides
-        dict.put("separate", conf.getBoolean("separate"))
-        dict.put("order", Consts.NEW_CARDS_DUE)
-        dict.put("perDay", mReportLimit)
-        return dict
+        return dict.apply {
```
Comment (divyanshdxn) [inline AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.kt:1926]: remove the local variable dict as well, something like `return JSONObject().apply{...}`

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.kt [hunk 2]
```diff
@@ -1941,18 +1943,18 @@ open class SchedV2(col: Collection) : AbstractSched(col) {
...
-        // original deck
-        dict.put("minInt", oconf.getJSONObject("lapse").getInt("minInt"))
-        dict.put("leechFails", oconf.getJSONObject("lapse").getInt("leechFails"))
-        dict.put("leechAction", oconf.getJSONObject("lapse").getInt("leechAction"))
-        dict.put("mult", oconf.getJSONObject("lapse").getDouble("mult"))
-        dict.put("delays", oconf.getJSONObject("lapse").getJSONArray("delays"))
-        // overrides
-        dict.put("resched", conf.getBoolean("resched"))
-        return dict
+        return dict.apply {
```
Comment (divyanshdxn) [inline AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.kt:1948]: same as earlier

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.kt (hunk 1 above)
Comment (criticalAY) [reply AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.kt:1926]: I left that for better readability but sure we can do that

**Comment by divyanshdxn** [comment]
LGTM

**Review by lukstbit — APPROVED** [review]

**Pull request merged**